cefbrowser/cefkeyboard.py

```diff
@@ -192,11 +192,6 @@ def get_windows_key_code(self, kivycode):
         #   OnPreKeyEvent info for key events and replacate it here.
         #   (key codes can also be found on MSDN Virtual-key codes page)
 
-        # Must map basic letters, otherwise eg. ctrl+a to select all
```

Comment:
Possibly you removed some other code and it broke. It works in kivy_.py example in cefpython.